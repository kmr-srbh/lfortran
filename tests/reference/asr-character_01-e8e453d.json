{
    "basename": "asr-character_01-e8e453d",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/warnings/character_01.f90",
    "infile_hash": "f952599c29229b99c5aca49cbb1e3ee4ff0dbd778b07ce826549ff4d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-character_01-e8e453d.stderr",
    "stderr_hash": "8aedba9723559e832c901f3a3848b777cdded0d78a6800e601ae0e6a",
    "returncode": 2
}