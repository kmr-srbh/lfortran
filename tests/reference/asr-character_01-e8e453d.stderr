semantic error: The RHS character len must be known at compile time
 --> tests/warnings/character_01.f90:3:5
  |
3 |     character(len=1, kind=1), parameter :: c_null_char = "ball"
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
